RELEASE-NOTES.md

```diff
@@ -16,6 +16,33 @@ See the following PRs for implementation details:
 - https://github.com/PrefectHQ/prefect/pull/11473
 - https://github.com/PrefectHQ/prefect/pull/11481
 
+### Get type-checked input from humans in the loop
+
+Human-in-the-loop flows just got an upgrade. You can now pause or suspend a flow and wait for type-checked input. To get started, declare the structure of the input data using a Pydantic model, and Prefect will render a form dynamically in the UI when a human resumes the flow. Form validation will ensure that the data conforms to your Pydantic model, and your flow will receive tbe input.
+
```

Comment:
@abrookins this might be worth adding to these release notes as well: 
https://github.com/PrefectHQ/prefect/releases/tag/2.14.12